### ISSUE TITLE
['TSDoc' home page]: Focus is not visible properly on 'Intro' link in 'TSDoc' home page.[Sev2][WCAG2.4.7]

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#18]
**Test Environment:**
OS Build: Windows 10
Browser: Edge Dev
URL: https://tsdoc.org/

**Repro Steps:**
1) Open the above URL and login with valid credentials.
2) TSDoc page will get displayed.
3) Navigate to 'Intro' link via keyboard and observe the issue.

**Actual:**
Keyboard focus is not visible properly on  'Intro' link in 'TSDoc' home page. .

**Expected:**
Focus indicator(s) must be clearly visible on each interactive element when navigating the page using the tab key.

**User Impact:**
Keyboard only users need to know which element has focus so they can track their navigation through and interaction with interactive controls. Note that non-interactive content will not get keyboard focus and cannot be navigated to with the keyboard alone. 

**Note:** Same issue is observed for 'Tags' , 'GitHub', 'Help' and 'Playground' links in home page.

**Observation:** Focus is not visible on 'Intro' , 'Tags' , 'GitHub', 'Help' and 'Playground' links in home page when High contrast is enabled. Screenshot is attached with initials _ Keyboard focus is not visible_High Contrast.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible

https://user-images.githubusercontent.com/85154417/121008895-1e836b00-c7b1-11eb-8f61-582f46bfb2e9.mp4
